webapi/src/main/scala/org/knora/webapi/responders/admin/PermissionsResponderADM.scala

```diff
@@ -646,12 +709,15 @@ class PermissionsResponderADM(responderData: ResponderData) extends Responder(re
     /**
      * Gets a single default object access permission identified by its IRI.
      *
-     * @param permissionIri the IRI of the default object access permission.
+     * @param permissionIri     the IRI of the default object access permission.
+     * @param requestingUser    the [[UserADM]] of the requesting user.
+     * @param apiRequestID      the API request ID.
      * @return a single [[DefaultObjectAccessPermissionADM]] object.
      */
-    private def defaultObjectAccessPermissionForIriGetRequestADM(permissionIri: IRI, requestingUser: UserADM): Future[DefaultObjectAccessPermissionForIriGetResponseADM] = {
-
-        // FIXME: Check user's permission for operation (issue #370)
+    private def defaultObjectAccessPermissionForIriGetRequestADM(permissionIri: IRI,
+                                                                 requestingUser: UserADM,
+                                                                 apiRequestID: UUID
+                                                                ): Future[DefaultObjectAccessPermissionForIriGetResponseADM] = {
```

Comment:
Still needs a check if the user is project admin in the project that the permission is about.

Reply:
this validation can be added to the messages in another PR as we have discussed above.